### DISCUSSION TITLE
How i can receive notification updates?

### DISCUSSION THREAD

**Discussion by conocimientotransformador** [SpiralLinux/SpiralLinux-project#163] [Q&A]
Hello, i want to have the notification updates, in automatically check for updates i put "once a day", but i still don't receiving notification updates, anyone knows the solution?

**Comment by robertocannito** [comment]
On debian sid I use https://packages.debian.org/buster/package-update-indicator
but just for notify, for update always:

sudo apt update
sudo apt upgrade

**Reply by geckolinux** [reply]
Thanks for that option @robertocannito .
I personally dislike update notifications. But another option is to open Gnome Software, and even after closing it will remain running and will show you an update notification popup about once every 24h. (I prefer Synaptic for updates, but Gnome Software does seem to work for updates if you don't mind them being installed during reboot. This is the recommended GUI method: https://github.com/SpiralLinux/SpiralLinux-project/wiki#updates-within-a-debian-stable-release-lifecycle)
